client/src/components/navbar/Navbar.js

```diff
@@ -93,15 +90,20 @@ function Navbar() {
         <p className="osu-logo">OSU CS Applied Plan Portal</p>
       </Link>
       <div className="right-container">
+        {role ? <History /> : null}
+        <Notifications />
         {role === 2 ? (
           <Link to={"/manageRoles"}>
             <button id="manage-roles-button">Manage Roles</button>
           </Link>
         ) : (
           null
         )}
-        {role ? <History /> : null}
-        <Notifications />
+        {role === 2 ? (
+          <UpdateCourses />
+        ) : (
+          null
+        )}
```

Comment:
![image](https://user-images.githubusercontent.com/14208546/78198983-3938b200-743f-11ea-9167-ce41377b82c6.png)

For some reasons, I couldn't see the `UpdateCourses` button even though I was logged in as a Head Advisor.

The console didn't throw any error.

Reply:
I'm dumb. I opened the up-to-date version of your branch remotely on WSL and ran the out-of-date version on my Windows system. It's working.